app/layout.tsx

```diff
@@ -19,17 +19,10 @@ export default function RootLayout({
   children: React.ReactNode;
 }) {
   return (
-    <html lang="en">
-      <body className="flex flex-col h-screen justify-between">
+    <html lang="en" className="!scroll-smooth">
```

Comment:
hmm what is this? Disable smooth scroll?

Reply:
It was... needed for something I tried to do but didn't end up implementing. I'll remove it.